lib/actions-oclif/os/versions.ts

```diff
@@ -54,13 +59,41 @@ export default class OsVersionsCmd extends Command {
 	public async run() {
 		const { args: params } = this.parse<FlagsDef, ArgsDef>(OsVersionsCmd);
 
-		const {
-			versions: vs,
-			recommended,
-		} = await getBalenaSdk().models.os.getSupportedVersions(params.type);
+		let versions = await this.getOsVersionsForDeviceType(params.type);
+		if (versions.length === 0) {
+			const aliases = await getDeviceTypeAliases();
+			if (aliases[params.type]) {
+				versions = await this.getOsVersionsForDeviceType(aliases[params.type]);
+			}
+		}
+		console.log(versions.join('\n'));
+	}
 
-		vs.forEach((v) => {
-			console.log(`v${v}` + (v === recommended ? ' (recommended)' : ''));
+	protected async getOsVersionsForDeviceType(
+		deviceType: string,
+	): Promise<string[]> {
+		const osUtils = await import('../../utils/os');
+		const allVersions = await osUtils.getAllOsVersions([deviceType]);
+		const versionObjects = allVersions[deviceType] || [];
+		return versionObjects.map((vObj) => {
+			let fv = vObj.formattedVersion;
```

Comment:
The modifications of `vObj.formattedVersion` in this function are quite hacky. It would make sense to modify `os.ts` (if that file was indeed to become part of the CLI), but I wanted to avoid it in order to preserve the ability to copy-and-paste `os.ts` from `balena-io/balena-ui`.